File: projects/Examples/node_example/files/read.js
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.on("close", function() {
    console.log("\nExit")
    process.exit(0)
});

const read = pos => {
  return new Promise(( resolve, reject ) => {
    rl.question(`Reading ${pos} number: `, function(number) {
      resolve( number )
    });
  });
}

File: projects/Examples/rust_example/files/read.rs
fn read (pos: &str) -> String {
  
  println!("Reading {} number:", pos);
  
  let mut number = String::new();
  io::stdin()
  .read_line(&mut number)
  .expect("failed to read from stdin");
  let a = String::from( number.trim() );
  a
}
